artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/management/ManagementHelper.java

```diff
@@ -84,6 +84,10 @@
 
    public static final SimpleString HDR_MESSAGE_ID = new SimpleString("_AMQ_Message_ID");
 
+   public static final SimpleString HDR_PROTOCOL_Name = new SimpleString("_AMQ_ProtocolName");
```

Comment:
static final field should be all upper case. HDR_PROTOCOL_NAME
key should be '_' separated _AMQ_Protocol_Name